src/dictionary/doc-template/index.hbs

```diff
@@ -1,177 +1,98 @@
-      <table class="propertiesTable">
-        <thead>
+import { Meta, Source, Story } from "@storybook/addon-docs/blocks";
+import ColorSwatch, { ColorSwatchWrapper } from "components/util/ColorSwatch";
+import Table, { TableBody, TableCell } from "components/util/Table"
+
+export const TableLayout = ({ data, headers }) => (
+    <Table>
+      <thead>
+        <tr>
+          {headers.map(header => (
+            <th>{header}</th>
+          ))}
+        </tr>
+      </thead>
+      <TableBody>
+        {data.map(row => (
           <tr>
-            <th style="min-width: 360px;" id="css">CSS var</th>
-            <th id="js">JS var</th>
-            <th id="value">value</th>
+            {row.map(item => (
+              <TableCell>{item}</TableCell>
+            ))}
           </tr>
-        </thead>
-        <tbody>
-          {{#each this.motion}}
-            <tr>
-              <td class="js-clipboard" data-clipboard-text="{{this.varNames.css}}" headers="css">{{this.varNames.css}}</td>
-              <td class="js-clipboard" data-clipboard-text="{{this.varNames.js}}" headers="js">{{this.varNames.js}}</td>
-              <td class="js-clipboard" data-clipboard-text="{{this.value}}" headers="value">{{this.value}}</td>
-            </tr>
-          {{/each}}
-        </tbody>
-      </table>
-    </section>
-
-  <script
-    src="https://code.jquery.com/jquery-1.12.4.min.js"
-    integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
-    crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
-  <script src="https://unpkg.com/clipboard@2.0.0/dist/clipboard.min.js"></script>
-  <script type="text/javascript">
-    function autorun() {
-      toastr.options.preventDuplicates = true;
-
-      var clipboard = new ClipboardJS('.js-clipboard');
-
-      clipboard.on('success', function(e) {
-        toastr.clear();
-        toastr.info(`Copied <strong style='color: #003366'>"${e.text}"</strong>`);
-      });
-      clipboard.on('error', function(e) {
-        toastr.clear();
-        toastr.error('Failed to copy to clipboard 😓')
-      });
-
-    }
-    if (document.addEventListener) document.addEventListener("DOMContentLoaded", autorun, false);
-    else if (document.attachEvent) document.attachEvent("onreadystatechange", autorun);
-    else window.onload = autorun;
-  </script>
-  </body>
-</html>
+        ))}
+      </TableBody>
+    </Table>
+)
+
+<Meta title="FDS|1 - Dictionary" />
+
+# Dictionary
+
+Using JS Vars: 
+`import FDS from '@cbinsights/fds/lib/dictionary/js/styleConstants'`
+`<CheckIcon color={FDS.COLOR_MINT} />`
+
+Using CSS Vars:
+`.foo { color: var(--color-mint) }`
+
+## Colors
+
+<Story name="Colors">
+  <ColorSwatchWrapper>
+    {{#each this.color}}
+      <ColorSwatch hex="{{this.values.hex}}" cssVar="{{this.varNames.css}}" jsVar="{{this.varNames.js}}" />
+    {{/each}}
+  </ColorSwatchWrapper>
+</Story>
+
+## Layout
```

Comment:
Looks like the heading should go inside of each story to render to the doc area

Reply:
You can't put Markdown inside these components unfortunately 🙁

Reply:
fair enough - just took another look and i think it's fine as is anyway

Reply:
It does render to docs though, it's just that if you click into the section, it will scroll past the title (same as our other stories for our components)